src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -129,8 +129,17 @@ def get_sync_status(self):
     @Endpoint(path='/sync-policy')
     @EndpointDoc("Get the sync policy")
     @ReadPermission
-    def get_sync_policy(self, bucket_name='', zonegroup_name=''):
+    def get_sync_policy(self, bucket_name='', zonegroup_name='', fetch_all_bucket= None):
```

Comment:
we'll want to also get the policy's from all zonegroups as well. so let's call this simply `all` and fetch everything at once if this is specified?
```suggestion
    def get_sync_policy(self, bucket_name='', zonegroup_name='', all=None):
```